alacritty/src/cli.rs

```diff
@@ -250,7 +250,14 @@ impl Options {
         config.window.dimensions = self.dimensions.unwrap_or(config.window.dimensions);
         config.window.position = self.position.or(config.window.position);
         config.window.title = self.title.or(config.window.title);
-        config.window.class = self.class.or(config.window.class);
+
+        if let Some(class) = self.class {
+            let parts : Vec<_> = class.split(',').collect();
+            config.window.class.instance = Some(parts[0].into());
+            if parts.len() >= 2 {
+                config.window.class.class = Some(parts[1].into());
+            }
+        }
```

Comment:
If we update the CLI, we also need to update the manpage and completions.

Reply:
It seems like the completions currently do not support --class at all, do you want to include that in this PR or should I open a separate issue?

Reply:
I'd rather it be separate.  There's nothing in particular to complete for the option argument, anyway (it's completely arbitrary).

Reply:
Created https://github.com/jwilm/alacritty/issues/2612 to track this.